resources/views/index.blade.php

```diff
@@ -1,12 +1,6 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Home</title>
-</head>
-<body>
-    <h1>Home</h1>
-</body>
-</html>
\ No newline at end of file
+@extends('layouts.app')
```

Comment:
do we want to use this syntax or the <x-> syntax (as in the footer branch)?

Reply:
From what I can tell the x-component still needs to have the html markup on every page? 🤔
While the include/yield templating only declares this once and then imports or extends the main layout 😃 

I might have missed something in the footer branch templating though... Let's take a closer look at this together during todays meeting so we can decide on which one we'll use moving forward 👍